test/reducers/filterReducerTest.js

```diff
@@ -33,24 +34,41 @@ const changedEventFilter = { incidentId: 1, ticketId: 0, eventTypes: [1, 2, 3] }
 
 describe('filterReducer', function () {
```

Comment:
In general, we should not test combined reducers; we should test each reducer individually. This makes the tests more focused and easy to update as the parts of the combined reducer are updated over time. For most of these reducers, you'd only need to test 3 cases:

* When CHANGE_EVENT_FILTER has a filter value, and the value is different from the state, return the filter value
* When CHANGE_EVENT_FILTER has a filter value, and the value is the same as the state, return the state
* When another action is received (I use { type:'IGNOREME' }), return the state.

You can write this test set once and copy it several times with minor adjustments to match each of the reducers.

That doesn't necessarily need to happen in this pull request, but it will need to happen eventually, as testing the combined reducers introduces much more complexity than testing the individual reducers, and that complexity grows sharply as we add cases to the individual reducers (especially when multiple parts of the combined reducer respond to the same action).